package.json

```diff
@@ -38,12 +39,11 @@
     "eslint": "3.19.0",
     "eslint-config-edx": "4.0.4",
     "eslint-config-edx-es5": "4.0.1",
-    "extract-text-webpack-plugin": "2.1.2",
```

Comment:
`extract-text-webpack-plugin` isn't supported in Webpack 4 so replaced it with `mini-css-extract-plugin`